src/components/Home/Home.tsx

```diff
@@ -22,16 +23,36 @@ export const Home: React.FunctionComponent = () => {
     { imageUrl: "images/homeImages/10-PlanetMars_1.jpg", imageInfo: "Civilisations throughout history have known Mars for its reddish colour, reminiscent of blood. The name Mars comes from the Ancient Roman God of War." }
   ];
 
+  function scrollNextImage() {
+    selectedImage < 9 ?
+      setSelectedImage(selectedImage + 1) :
+      setSelectedImage(0);
+  }
+
+  useEffect(() => {
+    const intervalId = setInterval(
+      () => {
+        scrollNextImage()
+      }, 3000);
+    return () => clearInterval(intervalId);
+  }, [selectedImage])
+
   return (
     <main>
       <h1 className="main-header">Mars Exploration</h1>
-      <p className="main-welcome-message">Start your 150 million km journey!</p>
-      <section className="main-image">
-        <img src={homeImages[selectedImage].imageUrl}></img>
-      </section>
-      {/* <section className="main-image-description"><p>{homeImages[selectedImage].imageInfo}</p></section>
-      <button onClick={() => { if (selectedImage < 9) { setSelectedImage(selectedImage + 1) } }}>Next</button>
-      <button onClick={() => { if (selectedImage > 0) { setSelectedImage(selectedImage - 1) } }}>Previous</button> */}
+      <p className="main-welcome-message">Start your 150 million km journey! </p>
+      <section className="image-info-section">
+        <article className="main-image">
+          <img className="main-image-tag" src={homeImages[selectedImage].imageUrl}></img>
+        </article>
+        <article className="main-image-description">
+          <p>{homeImages[selectedImage].imageInfo}</p>
+        </article>
+        <article className="left-right-home-buttons">
+          <button className="previous-left-button" onClick={() => { selectedImage > 0 ? setSelectedImage(selectedImage - 1) : setSelectedImage(0) }}><FaArrowCircleLeft className="button-icon" /></button>
+          <button className="next-right-button" onClick={() => { scrollNextImage() }}><FaArrowCircleRight className="button-icon" /></button>
+        </article>
```

Comment:
Can you move each attribute and the content onto a new line to make it easier to read?
```
<button className="previous-left-button" 
    onClick={() => { selectedImage > 0 ? setSelectedImage(selectedImage - 1) : setSelectedImage(0) }}> 
    <FaArrowCircleLeft className="button-icon" />
</button>
```

Reply:
made changes as part of Mm0.12 homepage startjourneybutton